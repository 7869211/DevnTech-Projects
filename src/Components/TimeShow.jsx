import React from 'react';
import '../Styles/Main.css'; 

const TimeShow=({time})=>{
  return (
    <div className="timeShow">
      <p>{time}</p>
    </div>
  );
}

export default TimeShow;
