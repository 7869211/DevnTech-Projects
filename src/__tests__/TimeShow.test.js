import React from 'react';
import { render, screen } from '@testing-library/react';
import TimeShow from '../Components/TimeShow';

describe('Time component', () => {
    beforeAll(() => {
        jest.useFakeTimers();
        jest.setSystemTime(new Date('27 Aug 2024 02:12:00 GMT').getTime())
      });

      afterAll(() => {
        jest.useRealTimers();
      });

    test('renders current time', () => {
        render(<TimeShow />);

       
    })
});