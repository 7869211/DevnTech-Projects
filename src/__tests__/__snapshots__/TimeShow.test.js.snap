// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeShow renders correctly 1`] = `
<DocumentFragment>
  <div
    class="timeShow"
  >
    <p>
      test
    </p>
  </div>
</DocumentFragment>
`;
